src/commands/sdm.js

```diff
@@ -70,26 +71,59 @@ exports.run = (client, message, args, callback) => {
     if (!args.length) return callback("Bruder da ist keine Frage :c");
 
     let question = args.join(" ").replace(/\s\s+/g, " ");
-    if (!question.endsWith("?")) question += "?";
+    const options = question.split(/,|;|\s+oder\s+/gi).map(s => s.trim()).filter(s => !!s);
 
-    const decision = !!secureDecisionMaker(question);
-    const image = decision ? "yes.png" : "no.png";
+    if(options.length > 1) {
+        question = options.reduce((p, c, i, a) => {
+            if(i === a.length - 1) {
+                return `${p} oder ${c}`;
+            }
+            return `${p}, ${c}`;
+        });
+    }
+    if (!question.endsWith("?")) question += "?";
 
     let embed = {
         embed: {
-            description: `**${question}**`,
+            title: question,
             timestamp: moment.utc().format(),
-            color: decision ? 0x2ecc71 : 0xe74c3c,
-            thumbnail: {
-                url: `attachment://${image}`
-            },
             author: {
                 name: `Secure Decision für ${message.author.username}`,
                 icon_url: message.author.displayAvatarURL()
             }
-        },
-        files: [`./assets/${image}`]
+        }
     };
+    if(options.length === 1) {
+        const decision = secureDecisionMaker(question);
+        let file;
+        if(!!decision) {
+            embed.embed.color = 0x2ecc71;
+            file = "yes.png";
+        }
+        else {
+            embed.embed.color = 0xe74c3c;
+            file = "no.png";
+        }
+        embed.embed.thumbnail = {
+            url: `attachment://${file}`
+        };
+        embed.files = [`./assets/${file}`];
+    }
+    else {
+        const decision = secureDecisionMaker(question, options.length - 1);
+        embed = {
+            embed: {
+                title: question,
+                description: `Mashallah, ich rate dir zu **${options[decision]}**!`,
+                timestamp: moment.utc().format(),
+                color: 0x9b59b6,
+                author: {
+                    name: `Secure Decision für ${message.author.username}`,
+                    icon_url: message.author.displayAvatarURL()
+                }
+            }
+        };
+    }
```

Comment:
Not a fan of that but whatever. 
Why not conditionally build the object rather than defining the object above and completely overriding it based on an expression?

Reply:
Because @diewellenlaenge stressed me 😢